exercises/Tests-R.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: exercise
+title: Tests
+language: R
+---
+
+This is a follow up to the [Debugging problem]({{ site.baseurl }}/exercises/Debugging-R).
+
+Write tests for your 'Debugging' code for the following cases and save it in a 
+file called `Tests-R.R`. Make any corrections/improvements that to the 
```

Comment:
"Make any corrections/improvements that to the" -> "Make corrections/improvements to the"